lib/services/systemd.c

```diff
@@ -171,7 +203,7 @@ systemd_loadunit_result(DBusMessage *reply, svc_action_t * op)
                                DBUS_TYPE_INVALID);
     }
 
-    if(op) {
+    if(op && path) {
```

Comment:
OK, that makes sense. But don't we still need to finalize the operation if path==NULL at this point?